core/standard/clause_7_building_blocks.adoc

```diff
@@ -909,6 +911,14 @@ sort order.
 
 include::requirements/sorting/REQ_defaultSortOrder-definition.adoc[]
 
+==== Sorting by relevance (informative)
+
+A common practice is to sort search results by relevance.
+
+This capability is accomodated in this Standard by using <<per_sorting_sortables-synthetic,synthetic>> sortables.  That is, sortables that are derived or synthesized by the server.
```

Comment:
-> This capability is accommodated in this Standard by using <<per_sorting_sortables-synthetic,synthetic>> sortables.  That is, sortables that are derived or synthesized at the server's discretion.

Reply:
I am ok to leave as is